query_execution/QueryExecutionTypedefs.hpp

```diff
@@ -98,6 +98,12 @@ enum QueryExecutionMessageType : message_type_id {
 #endif
 };
 
+// WorkOrder profiling data structures.
+// A tuple of <Worker ID, Operator ID, Start Time, End Time>
+typedef std::tuple<std::size_t, std::size_t, std::size_t, std::size_t> WorkOrderTimeEntry;
```

Comment:
It may be helpful to make WorkOrderTimeEntry as a struct with the parameters as public variables. That way we will be less concerned about the ordering of the parameters and any potential bugs.